hal/src/main/native/athena/FRCDriverStation.cpp

```diff
@@ -138,7 +138,7 @@ static int32_t HAL_GetJoystickButtonsInternal(int32_t joystickNum,
 static int32_t HAL_GetJoystickDescriptorInternal(int32_t joystickNum,
                                                  HAL_JoystickDescriptor* desc) {
   desc->isXbox = 0;
-  desc->type = std::numeric_limits<uint8_t>::max();
+  desc->type = (std::numeric_limits<uint8_t>::max)();
```

Comment:
Ugh.  Can we disable this warning globally please and not make these changes to min/max?  It is very non standard C++ to have to write things this way.

Reply:
This one's really hard. With the tools I was doing this for, there is no way to default define anything. So without this, things will randomly break in other projects.